includes/site-health/audit-autoloaded-options/hooks.php

```diff
@@ -26,3 +26,86 @@ function perflab_aao_add_autoloaded_options_test( $tests ) {
 	return $tests;
 }
 add_filter( 'site_status_tests', 'perflab_aao_add_autoloaded_options_test' );
+
+/**
+ * Register admin actions for handling autoload enable/disable.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_register_admin_actions() {
+	add_action( 'admin_action_perflab_aao_update_autoload', 'perflab_aao_handle_update_autoload' );
+}
+add_action( 'admin_init', 'perflab_aao_register_admin_actions' );
+
+/**
+ * Callback for handling disable autoload action.
+ *
+ * @since n.e.x.t
+ */
+function perflab_aao_handle_update_autoload() {
+	// Verify nonce.
+	$nonce = isset( $_GET['_wpnonce'] ) ? sanitize_text_field( $_GET['_wpnonce'] ) : '';
+
+	if ( ! wp_verify_nonce( $nonce, 'perflab_aao_update_autoload' ) ) {
+		wp_die( esc_html__( 'Nonce verification failed.', 'performance-lab' ) );
+	}
+
+	$option_name = isset( $_GET['option_name'] ) ? sanitize_text_field( $_GET['option_name'] ) : '';
+	$autoload    = isset( $_GET['autoload'] ) ? sanitize_text_field( $_GET['autoload'] ) : '';
+	$value       = isset( $_GET['value'] ) ? sanitize_text_field( $_GET['value'] ) : '';
+
+	if ( ! current_user_can( 'manage_options' ) ) {
+		wp_die( esc_html__( 'Permission denied.', 'performance-lab' ) );
+	}
+
+	if ( empty( $option_name ) || empty( $autoload ) ) {
+		wp_die( esc_html__( 'Invalid option name or autoload value.', 'performance-lab' ) );
+	}
+
+	// Check if the option exists.
+	if ( false === get_option( $option_name ) ) {
+		// Option doesn't exist, return an error or handle the situation accordingly.
+		wp_die( esc_html__( 'The option does not exist.', 'performance-lab' ) );
+	}
+
+	$result = wp_set_option_autoload( $option_name, $autoload );
```

Comment:
Looks like Performance Lab requires WP 6.3+ but this function has only been added in 6.4.

We either should bump our requirement or add some safeguards.

Reply:
Per the policy outlined in https://make.wordpress.org/performance/handbook/performance-lab/version-support-policy/#wordpress-core-versions, Performance Lab is committed to supporting the latest stable version of WordPress core. Additionally, considering that this change is expected to be released in version 3.0, which is slated for mid-April, and WP 6.5 is scheduled for release on March 26, we propose bumping the minimum version for Performance Lab to 6.4.

What do you think? cc. @felixarntz

Reply:
Open followup issue #1056

Reply:
Do it. 🚀

Reply:
Replied on the issue, SGTM!